api-v1.md

```diff
@@ -173,4 +173,27 @@ If no errors were encountered during compilation, the provided error array is em
         },
     ]
 }
-```
\ No newline at end of file
+```
+
+# GET /lobby/{lobbyId}/settings
+Gets the settings of a lobby.
+
+## responses:
+### 200:
+```json
+{
+    "settings": {
+        "coreSize": number,
+        "instructionLimit": number,
+        "initialInstruction": String,
+        "maximumTicks": number,
+        "maximumProcessesPerPlayer": number,
+        "readDistance": number,
+        "writeDistance": number,
+        "minimumSeparation": number,
+        "separation": number,
+        "randomSeparation": boolean,
```

Comment:
I'm wondering, should we split this up into separate lobby settings and interpreter settings to logically separate them and make it easier to work with / extend in the future?
Something like
```json
{
    "settings:": {
        // Possibly later lobbySettings could just be added
        "lobbySettings" : {},
        "interpreterSettings": {
            "coreSize": number,
            "instructionLimit": number,
            "initialInstruction": String,
            "maximumTicks": number,
            "maximumProcessesPerPlayer": number,
            "readDistance": number,
            "writeDistance": number,
            "minimumSeparation": number,
            "separation": number,
            "randomSeparation": boolean,
        }
    },
}
```

Reply:
:100: